plugins/modules/discovery_intent.py

```diff
@@ -1060,6 +1061,10 @@ def preprocess_device_discovery(self, ip_address_list=None):
             else:
                 self.preprocess_device_discovery_handle_error()
         else:
+            if len(ip_address_list) > 8:
+                msg = "MaAttempt to use more than 8 IP ranges detected. The system allows a maximum of 8."
```

Comment:
Attempt to use more than 8 IP ranges detected. The system allows a maximum of 8.

Reply:
Addressed